datafusion/sqllogictest/test_files/explain_tree.slt

```diff
@@ -1733,21 +1734,67 @@ physical_plan
 01)┌───────────────────────────┐
 02)│  SortPreservingMergeExec  │
 03)│    --------------------   │
-04)│   time@2 ASC NULLS LAST,  │
-05)│    date@0 ASC NULLS LAST  │
-06)└─────────────┬─────────────┘
-07)┌─────────────┴─────────────┐
-08)│    CoalesceBatchesExec    │
-09)│    --------------------   │
-10)│     target_batch_size:    │
-11)│            8192           │
-12)└─────────────┬─────────────┘
-13)┌─────────────┴─────────────┐
-14)│         FilterExec        │
-15)│    --------------------   │
-16)│         predicate:        │
-17)│ ticker@1 = A AND CAST(time│
-18)│   @2 AS Date32) = date@0  │
+04)│    date ASC NULLS LAST    │
+05)│                           │
+06)│   time ASC NULLS LAST,    │
+07)└─────────────┬─────────────┘
+08)┌─────────────┴─────────────┐
+09)│    CoalesceBatchesExec    │
+10)│    --------------------   │
+11)│     target_batch_size:    │
+12)│            8192           │
+13)└─────────────┬─────────────┘
+14)┌─────────────┴─────────────┐
+15)│         FilterExec        │
+16)│    --------------------   │
+17)│         predicate:        │
+18)│  ticker = A AND CAST(time │
```

Comment:
this looks so nice

![giphy](https://github.com/user-attachments/assets/037d980d-11c7-4df3-8027-2b95f3bb4900)

(I am using that image a log recently 🤔 )